src/mon/PGMap.cc

```diff
@@ -2886,7 +2886,7 @@ int reweight::by_utilization(
     for (const auto& pg : pgm.pg_stat) {
       if (pools && pools->count(pg.first.pool()) == 0)
 	continue;
-      for (const auto acting : pg.second.acting) {
+      for (const auto acting : pg.second.up) {
```

Comment:
i wonder, if it is the **acting** set who is serving the pg and they are where the objects of this PG are currently stored. why shall we reweight the **up** set based on the utilization of the up set who is **not** holding the objects of the PG? this does sound reasonable to me.